homer0/projext-plugin-webpack
# feat(project): watch files involved in the creation of a browser target configuration

### What does this PR do?

> This is the implementation of homer0/projext#82.

It watches the files involved on the creation of a "browser target configuration", so if they're modified, webpack will reload the bundle.

### How should it be tested manually?

In order to test this, you need to install `homer0/projext#next` and make sure you delete the copy of `projext` `npm`/`yarn` will install inside this package's `node_modules`.

Create a browser target that uses a configuration file and try updating it while projext runs the target, the bundle should be reloaded.

And, of course...

```bash
yarn test
# or
npm test
```
